src/lib/database/seed.ts

```diff
@@ -99,6 +99,14 @@ export const seed = (db: Database): Seed => {
     },
   })
 
+  db.addDevice({
+    device_id: "noiseaware_device_1",
+    device_type: "noiseaware_activity_zone",
+    name: "Noise Sensor 1",
+    connected_account_id: "john_connected_account_id",
+    workspace_id: "seed_workspace_1",
+  })
+
```

Comment:
I think there aren't alot of noiseaware devices or we're not actively supporting it, could this be a minut device?

Reply:
https://github.com/seamapi/fake-seam-connect/pull/121/commits/8bd3103e64336aab359d4ba0925ea3958875ce7b